src/Algebra/Module/Morphism/LeftSemimoduleMonomorphism.agda

```diff
@@ -0,0 +1,147 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Consequences of a monomorphism between left semimodules
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Algebra.Module.Bundles.Raw
+open import Algebra.Module.Morphism.Structures
+
+module Algebra.Module.Morphism.LeftSemimoduleMonomorphism
+  {r a b ℓ₁ ℓ₂} {R : Set r} {M₁ : RawLeftSemimodule R a ℓ₁} {M₂ : RawLeftSemimodule R b ℓ₂} {⟦_⟧}
+  (isLeftSemimoduleMonomorphism : IsLeftSemimoduleMonomorphism M₁ M₂ ⟦_⟧)
+  where
+
+open IsLeftSemimoduleMonomorphism isLeftSemimoduleMonomorphism
+private
+  module M = RawLeftSemimodule M₁
+  module N = RawLeftSemimodule M₂
```

Comment:
Elsewhere you use `M` and `N` where here you have `M₁` and `M₂`?
Does it make sense to (try to) standardise on one or the other throughout this PR?